pkg/stub/handler.go

```diff
@@ -559,38 +552,47 @@ func (h *Handler) processFiles(dir string, files []os.FileInfo, opcfg *v1alpha1.
 	return nil
 }
 
+func (h *Handler) coreUpdateDockerPullSpec(oldreg, newreg string, oldies []string) string {
+	// see if the imagestream on file (i.e. the openshift/library content) is
+	// of the form "reg/repo/img" or "repo/img"
```

Comment:
since our centos imagestreams probably are of the form "openshift/myimage" (ie no docker.io prefix) i think this logic is necessary/good.  

it's also nicely encapsulated now, so +1 from me.

Reply:
ok cool thx